src/Swarrot/Broker/MessageProvider/SqsMessageProvider.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Swarrot\Broker\MessageProvider;
+
+use Aws\Sqs\SqsClient;
+use Swarrot\Broker\Message;
+use Swarrot\Driver\MessageCacheInterface;
+use Swarrot\Driver\PrefetchMessageCache;
+
+/**
+ * Class SqsMessageProvider.
+ */
+class SqsMessageProvider implements MessageProviderInterface
+{
+    protected $cache;
+    protected $channel;
+    protected $prefetch;
+    protected $queueName;
```

Comment:
@odolbeau I suggest using private properties. It makes it much easier to manage backward compatibility